### ISSUE TITLE
Adopt the change of the extensionKind format

### ISSUE THREAD

**Issue by egamma** [CoenraadS/Bracket-Pair-Colorizer-2#205]
As of VS Code 1.40 the extensionKind attribute in the package.json of the extension can be an array, pls see the [documentation](https://code.visualstudio.com/api/advanced-topics/remote-extensions#incorrect-execution-location).

In your current extension the old format `"extensionkind: "ui"` is used. We will be deprecating the string type for the extensionKind property in favour of string array type.

This extension must run as a "ui" extension locally since it accesses `vscode.env.appRoot` and loads the native textmate module. Therefore the extensionKind must be set to `"extensionkind: ["ui"]` indicating that it cannot run as a workspace extension.

**Comment by yichengsu** [comment]
Well, I still don't know how to use it in Remote-SSH.

**Comment by egamma** [comment]
@yichengsu there are issues with the coloring for python and json files that will be fixed on the VS Code side. Otherwise the extension should just work. When you have the extension installed locally, then it should automatically work for in the remote-ssh setup. If it doesn't work then please provide more details in a separate issue.

**Comment by parkerault** [comment]
I left a comment on a separate but duplicate issue, but if anyone is having trouble getting the extension to work on remote sessions, it works for me only after I open the vs code preferences window. I'm not sure why, but when I open the preferences and tab back to the editor, the brackets are colorized.
